zeq/detail/registerVocabulary.fbs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2015, Human Brain Project
+//                     Grigori Chevtchenko <grigori.chevtchenko@epfl.ch>
+
+namespace zeq.vocabulary;
+
+table RegisterVocabulary
+{
+  restNames:[string];
+  eventLows:[ulong];
+  eventHighs:[ulong];
+  schemas:[string];
+}
```

Comment:
Actually, shouldn't these be a vector of structs (struct: all data for one event)?